spec/elastic_apm/grape_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+if defined?(Grape)
+  require 'elastic_apm/grape'
```

Comment:
Wouldn't this already be required by the agent if defined?

Reply:
Not with the way the gems were being listed in the Gemfile - when running `Bundler.require`, the agent was being required before the frameworks. I've now made [this change](https://github.com/elastic/apm-agent-ruby/pull/562/commits/f81df980351c9bd9a2ccb93982efc4894868e7bf#diff-8b7db4d5cc4b8f6dc8feb7030baa2478) so requiring the framework modules in tests is no longer necessary.